rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -211,7 +232,49 @@ impl FirmwareProcessor {
                         txn.start_txn().send_response(&capabilities.to_bytes())?;
                         continue;
                     }
+
+                    CommandId::STASH_MEASUREMENT => {
+                        if measurement_count == MEASUREMENT_MAX_COUNT {
+                            cprintln!(
+                                "[afmc] Maximum supported number of measurements already received, ignoring."
+                            );
+                            txn.start_txn().complete(false)?;
+                            continue;
+                        }
+
+                        // Re-borrow mailbox to work around https://github.com/rust-lang/rust/issues/54663
```

Comment:
This isn't necessary; txn isn't long-lived here.

Reply:
Updated